libs/langchain-anthropic/src/tests/anthropic.test.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from "@jest/globals";
+
+// This is required while there are no (non integration) tests
+// inside this package. Otherwise, jest will complain about
+// "No tests found, exiting with code 1"
```

Comment:
Is there a more elegant way to avoid the error?

Reply:
Adding an actual test haha

Reply:
will investigate

Reply:
Yeah we can just import + instantiate a model or something